### ISSUE TITLE
About CRITIC_ITERS

### ISSUE THREAD

**Issue by caiyu6666** [dbbbbm/f-AnoGAN-PyTorch#7]
Thanks for your code, but I don't understand what's the function of the CRITIC_ITER. It seems that only the first 5 batches datas (CRITIC_ITER=5) are used for training. Could you please help explain it? Thank you.

**Comment by dbbbbm** [comment]
The CRITIC_ITER is used to train discriminator more. In other words, in every iteration the discriminator is trained by first 5 batches data. But the dataset is shuffled by pytorch's dataloader class, so the batches are different in each iteration.

**Comment by caiyu6666** [comment]
I get your point. Thank you!
